app/filters.js

```diff
@@ -1,4 +1,3 @@
-const moment = require('moment');
 
 module.exports = function (env) {
```

Comment:
Don't need to pass in `env` I don't think?

Reply:
This is the default construct for `app/filters.js`. I will leave as is and we can review at another time. I will create a ticket.